web/cobrands/fixmystreet/fixmystreet.js

```diff
@@ -12,6 +12,22 @@ function isR2L() {
     return !!$('html[dir=rtl]').length;
 }
 
+function make_multi(id) {
```

Comment:
I'm not massively keen on introducing a top-level function, could it be put somewhere with a narrower scope?

It'd work well in the 'small jQuery extensions' bit just below here (you'd have to get rid of the `id` param and use `this` instead), then it could be called as `$('#statuses').make_multi();` etc.